microsoft/runbook/examples/git_bisect.yml

```diff
@@ -0,0 +1,118 @@
+name: git_bisect
+extension:
+  - "../../testsuites"
+
+include:
+  - path: ../tiers/tier.yml
+  - path: ../azure.yml
+
+variable:
+  - name: subscription_id
+    value: ""
+  - name: tier
+    value: 0
+  - name: test_case_name
+    value: "smoke_test"
+  - name: marketplace_image
+    value: "canonical ubuntuserver 18.04-lts latest"
+  - name: repo_url
+    value: "git://git.kernel.org/pub/scm/linux/kernel/git/stable/linux-stable.git"
+  - name: good_commit
+    value: ""
+  - name: bad_commit
+    value: ""
+
+  # Do not assign values to below variables. These are used by the combinator.
+  - name: build_vm_address
+    value: ""
+  - name: bisect_vm_address
+    value: ""
+  - name: build_vm_resource_group_name
+    value: ""
+  - name: bisect_vm_resource_group_name
+    value: ""
+  - name: vhd
+    value: ""
+  - name: kernel_installer_is_success
+    value: False
+  - name: script_test_case_name
+    value: ""
+  - name: ref
+    value: ""
+
+transformer:
+  - type: azure_deploy
+    name: bisect_vm
+    requirement:
+      azure:
+        marketplace: $(marketplace_image)
+        location: $(location)
+      core_count: 2
+    enabled: true
+  - type: azure_deploy
+    phase: expanded
+    name: build_vm
+    requirement:
+      azure:
+        marketplace: $(marketplace_image)
+        location: $(location)
+      core_count: 16
+    enabled: true
+  - type: kernel_installer
+    phase: expanded
+    connection:
+      address: $(build_vm_address)
+      private_key_file: $(admin_private_key_file)
+    installer:
+      type: source
+      location:
+        type: repo
+        path: /mnt/code
+        ref: $(ref)
+        repo: $(repo_url)
+    raise_exception: False
+  # Do not create vhd when build fails
+  - type: azure_vhd
+    enabled: $(kernel_installer_is_success)
+    phase: expanded
+    resource_group_name: $(build_vm_resource_group_name)
+    rename:
+      azure_vhd_url: vhd
+  # Do not run test when build fails
+  - type: script
```

Comment:
It can reuse the $(kernel_installer_is_success) variable. It doesn't need to calculate.

Reply:
I did not understand.
Are you suggesting to keep default value for test_case_name as "" and set it to correct value by using `enabled: kernel_installer_is_success` instead of if else condition in script? `script: "test_case_name if kernel_installer_is_success else ''"`

Reply:
Currently I am including tiers.yml
Should I add the enabled property to t0.yml, t1.yml etc?
Or is it better to define the test case block within the sample runbook?


`
testcase:
  - criteria:
      name: $(test_case_name)
enabled: $(kernel_installer_is_success) 
`

Reply:
https://github.com/microsoft/lisa/pull/3014/files